contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -129,6 +131,19 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     return SafeCast.toUint96((1e18 * gasPriceWei) / getWeiPerUnitLink());
   }
 
+  function _getL1FeeJuels() private view returns (uint96) {
+    // Will return non-zero on chains that have this enabled
+    uint256 l1CostWei = ChainSpecificUtil._getCurrentTxL1GasFees(msg.data);
+
+    if (l1CostWei == 0) {
+      return 0;
+    }
+
+    // (1e18 juels/link) / (wei/link) = juels per wei
+    uint256 juelsPerWei = (1e18 / getWeiPerUnitLink());
```

Comment:
This is incorrect. We also have similar logic in _getJuelsPerGas() already so maybe we can combine it somehow?

Reply:
Sorry, this is correct, my bad.

Still I wonder if there's an opportunity to convert to juels only once.